user.js

```diff
@@ -633,7 +633,7 @@ module.exports = function user (options) {
       return done(null, {ok: false, why: why, user: user})
     }
 
-    if (user.lockTry >= options.lockTry && !_.isNull(options.lockTry)) {
+    if (options.failedLoginCount && user.failedLoginCount >= options.failedLoginCount) {
       seneca.log.debug('login/fail', why = 'locked-out', user)
       return done(null, {ok: false, why: why, user: user})
```

Comment:
Here the user is returned as part of the response. Because the `ok: false` I think user should not be returned.